app/models/katello/erratum.rb

```diff
@@ -86,7 +86,7 @@ def hosts_applicable(org_id = nil)
     end
 
     def hosts_available(org_id = nil)
-      self.hosts_applicable(org_id).joins("INNER JOIN #{Katello::RepositoryErratum.table_name} on \
+      self.hosts_applicable(org_id).distinct.joins("INNER JOIN #{Katello::RepositoryErratum.table_name} on \
```

Comment:
@hao-yu would you be comfortable adding a test for this?

Reply:
Yup, I can do that

Reply:
Somehow the pull request not getting updated. I will submit a new one.